### PR TITLE
[Quest] Ducal Hospitality

### PR DESCRIPTION
**Pull request by jamesbradleym**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Implement's quest Ducal Hospitality from [ASB Commit](https://github.com/AirSkyBoat/AirSkyBoat/commit/314d11c49e6c2ecca274dd079418da3160511ff7) with bug fix for duplicate first reward.

## Steps to test these changes

- Speak with Taillegeas in Ru-Lude Gardens with >= 4 Jueno fame and A Vessel Without a Captain completed.
- Find requested items
- Trade in items
- ??
- Profit

([Previously](https://github.com/LandSandBoat/server/pull/5337))

### CODE REVIEW

File: scripts/quests/jeuno/Ducal_Hospitality.lua [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Taillegeas'] =
+            {
+                onTrigger = function(player, npc)
+                    local warriorsPathComplete = player:hasCompletedMission(xi.mission.log_id.COP, xi.mission.id.cop.THE_WARRIORS_PATH) and 1 or 0
+                    local questItemSet = math.random(#questItemSets)
+                    quest:setVar(player, 'ItemSet', questItemSet)
+                    return quest:progressEvent(10057, { [0] = warriorsPathComplete,
```
Comment (zach2good) [inline scripts/quests/jeuno/Ducal_Hospitality.lua]: These probably want to be formatted closer to this:
```lua
return quest:progressEvent(10057, {
    [0] = warriorsPathComplete,
    [1] = questItemSets[questItemSet].items[1],
    [2] = questItemSets[questItemSet].items[2],
    [3] = questItemSets[questItemSet].items[3],
    [4] = questItemSets[questItemSet].items[4] or 0,
    [5] = questItemSets[questItemSet].items[5] or 0,
    [6] = questItemSets[questItemSet].textOption,
})
```

**Review by zach2good — APPROVED** [review]
Seems fine to me, just needs a formatting tweak then probably good to go.

(I got caught by accident on this quest once, it is as simple as it seems - funny turban guy, right?)

File: scripts/quests/jeuno/Ducal_Hospitality.lua (hunk 1 above)
Comment (jamesbradleym) [reply scripts/quests/jeuno/Ducal_Hospitality.lua]: Done :)

Comment (zach2good) [reply scripts/quests/jeuno/Ducal_Hospitality.lua]: All the other examples of the previous styling are still untouched

Comment (jamesbradleym) [reply scripts/quests/jeuno/Ducal_Hospitality.lua]: Whoops 😅 Done

**Review by zach2good — APPROVED** [review]

**Pull request merged**